git/gitRemind.go

```diff
@@ -0,0 +1,215 @@
+package git
```

Comment:
Why does Github show that these are all changed? I thought these files were in your git remind branch?

Reply:
They are, I couldn't figure out why they're showing up here either. I thought I branched from master but I may have branched from the GitRemind branch by accident, in which case I think they'd get added as changes to this branch too. I'll merge that one before I merge this one either way.

Reply:
I wouldn't worry about this too much. Seems like a github thing